packages/web-shared/components/FeatureFeed/Features/ScriptureFeature.js

```diff
@@ -21,16 +21,37 @@ function ScriptureFeature(props = {}) {
   };
 
   function parseBibleReference(reference) {
-    const regex = /^([\w\s]+)\s(\d+):(\d+)-?(\d+)?$/;
+    // This regex handles cases like 'Genesis 1-3' and 'Genesis 1:1-3:24'
+    const regex = /^([\w\s]+)\s(\d+)(?::(\d+))?(?:-(\d+)(?::(\d+))?)?$/;
     const match = reference.match(regex);
 
     if (!match) {
       return null; // Invalid format
     }
 
-    const [_, book, chapter, startVerse, endVerse] = match;
-    const title = `${book} ${chapter}`;
-    const verses = `${endVerse ? `Verses ${startVerse}-${endVerse}` : `Verse ${startVerse}`}`;
+    const [_, book, startChapter, startVerse, endChapter, endVerse] = match;
+    let title, verses;
+
+    if (endChapter || endVerse) {
+      // Case for a range
+      if (endChapter && !endVerse) {
+        // Range of chapters, e.g., 'Genesis 1-3'
+        title = `${book} ${startChapter}-${endChapter}`;
+        verses = `Chapters ${startChapter}-${endChapter}`;
+      } else if (startChapter !== endChapter) {
+        // Range of chapters and verses, e.g., 'Genesis 1:1-3:24'
+        title = `${book} ${startChapter}:${startVerse}-${endChapter}:${endVerse}`;
+        verses = `Verses ${startChapter}:${startVerse}-${endChapter}:${endVerse}`;
```

Comment:
small nitpik - for clarity that we reuse this text :D 
```suggestion
        verses = `Verses ${title}`;
```

Reply:
<img width="449" alt="Screenshot 2024-01-31 at 4 19 11 PM" src="https://github.com/ApollosProject/apollos-embeds/assets/72768221/845779a6-5873-4806-9a93-3e0b49fbb089">

Like this?

Reply:
🤦 I missed that the second sample doesn't include the book. What you have is perfect